### ISSUE TITLE
Doesn't work on nested declarations in SCSS?

### ISSUE THREAD

**Issue by danielbayley** [1000ch/atom-csscomb#7]
It won't work on the following code for example.

```
nav {
  ul {
    margin: 0;
    padding: 0;
    list-style: none;
  }

  li { display: inline-block; }
  //Comment
  a {
    display: block;
    padding: 6px 12px;
    text-decoration: none;
  }
}
```

**Comment by 1000ch** [comment]
Sorry for delay. This problem is fixed in `v1.2.1`, so please try this version :)

**Comment by danielbayley** [comment]
Seems to work :+1:  but I've switched to [css-comb](https://atom.io/packages/css-comb) as it works best out of all the css comb packages and is more configurable.
